### PR TITLE
Feat: Add AutoMuteUs option to fix an issue with the Host Name

### PR DESCRIPTION
**Pull request by GayKevin**
Feat: Add AutoMuteUs option to fix an issue with the Host Name

### CODE REVIEW

**Comment by NikoCat233** [comment]
Sorry but i cant get the point of this...

**Comment by GayKevin** [comment]
> Uhh you want to force host not load custom names? Then what issue does this fix
> 
> And you should not put the check here if u want to completely prevent host from using custom names

The host of the game is getting special characters on his name. It makes the bot falling to match the users and to apply the correct rules for it.
It doesn't happen with the other mods that are using the same code base except for this part

**Comment by NikoCat233** [comment]
Then it can not be done simply with changing host's name...
We also modify clients' names under some condition.

@Tommy-XL  Should we add support for automuteus? https://www.youtube.com/watch?v=VYx6kM1O4FM

**Comment by Tommy-XL** [comment]
> Then it can not be done simply with changing host's name... We also modify clients' names under some condition.
> 
> @Tommy-XL Should we add support for automuteus? https://www.youtube.com/watch?v=VYx6kM1O4FM

Why not? lol

**Comment by NikoCat233** [comment]
This commit prevent showing host's custom name in lobby LOL
I wonder how the bot works in game.

File: Modules/Utils.cs [hunk 1]
```diff
@@ -1729,7 +1729,7 @@ public static void ApplySuffix(PlayerControl player)
             {
                 if (!player.IsModClient()) return;
                 {
-                    if (GameStates.IsOnlineGame || GameStates.IsLocalGame)
+                    if ((GameStates.IsOnlineGame || GameStates.IsLocalGame) && Main.AutoMuteUs.Value != true)
```
[the remark this answers is not in the document]
Comment (Tommy-XL) [reply Modules/Utils.cs:1732]: I'll fix it myself, since it's not critical

**Pull request merged**